_pytest/doctest.py

```diff
@@ -94,7 +111,7 @@ def repr_failure(self, excinfo):
             message = excinfo.type.__name__
             reprlocation = ReprFileLocation(filename, lineno, message)
             checker = _get_checker()
-            REPORT_UDIFF = doctest.REPORT_UDIFF
+            REPORT_UDIFF = _get_report_choice(self.config.getoption("doctestreport"))
```

Comment:
Heh we missed the variable name `REPORT_UDIFF`... should be probably `report_choice` or so.